### PR TITLE
DeformableConvolution v8: ngraph python API

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
 - *Python API for Deformable convolution v8"
 - *unit tests*

### Tickets:
 - *56590*

### CODE REVIEW

File: ngraph/python/src/ngraph/utils/node_factory.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 from ngraph.impl import Node, Output
 
-DEFAULT_OPSET = "opset6"
+DEFAULT_OPSET = "opset8"
```
Comment (jiwaszki) [inline ngraph/python/src/ngraph/utils/node_factory.py:11]: Also this change should be also in separate PR.

File: ngraph/python/src/ngraph/opset8/__init__.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from ngraph.opset1.ops import absolute
```
Comment (jiwaszki) [inline ngraph/python/src/ngraph/opset8/__init__.py:4]: Also these changes (whole file) should be also in separate PR.

File: ngraph/python/src/pyngraph/node_factory.cpp [hunk 3]
```diff
@@ -75,6 +75,7 @@ namespace
                 {"opset5", OpsetFunction(ngraph::get_opset5)},
                 {"opset6", OpsetFunction(ngraph::get_opset6)},
                 {"opset7", OpsetFunction(ngraph::get_opset7)},
+                {"opset8", OpsetFunction(ngraph::get_opset8)},
```
Comment (jiwaszki) [inline ngraph/python/src/pyngraph/node_factory.cpp:78]: Also this change should be also in separate PR.

File: ngraph/python/src/ngraph/opset8/ops.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                "strides": strides,
+                "pads_begin": pads_begin,
+                "pads_end": pads_end,
+                "dilations": dilations,
+                "auto_pad": auto_pad,
+                "group": group,
+                "deformable_group": deformable_group,
+                "bilinear_interpolation_pad": bilinear_interpolation_pad
+            },
+        )
```
Comment (jiwaszki) [inline ngraph/python/src/ngraph/opset8/ops.py:112]: This is correct, however I would like to see solution similar to (just for example) `non_max_suppression` or `dft` ops. Always return Node but change inputs and attributes accordingly to None values.

**Review by jiwaszki — CHANGES_REQUESTED** [review]
Please separate opset8 and new op in two PR. Besides that looks fine to me 👍

File: ngraph/python/src/ngraph/__init__.py [hunk 5]
```diff
@@ -28,159 +28,159 @@
...
-from ngraph.opset7 import swish
-from ngraph.opset7 import tan
-from ngraph.opset7 import tanh
-from ngraph.opset7 import tensor_iterator
-from ngraph.opset7 import tile
-from ngraph.opset7 import topk
-from ngraph.opset7 import transpose
-from ngraph.opset7 import unsqueeze
-from ngraph.opset7 import variadic_split
+from ngraph.opset8 import absolute
```
[the remark this answers is not in the document]
Comment (itikhono) [reply ngraph/python/src/ngraph/__init__.py:31]: PR:  https://github.com/openvinotoolkit/openvino/pull/6535/

File: ngraph/core/src/opsets/opset.cpp [hunk 6]
```diff
@@ -115,3 +115,15 @@ const ngraph::OpSet& ngraph::get_opset7()
...
+{
+    static OpSet opset;
+    static std::once_flag flag;
+    std::call_once(flag, [&]() {
+#define NGRAPH_OP(NAME, NAMESPACE) opset.insert<NAMESPACE::NAME>();
+#include "ngraph/opsets/opset8_tbl.hpp"
+#undef NGRAPH_OP
+    });
+    return opset;
+}
```
[the remark this answers is not in the document]
Comment (itikhono) [reply ngraph/core/src/opsets/opset.cpp:129]: fixed in https://github.com/openvinotoolkit/openvino/pull/6535/

File: ngraph/python/src/ngraph/utils/node_factory.py (hunk 1 above)
Comment (itikhono) [reply ngraph/python/src/ngraph/utils/node_factory.py:11]: done

File: ngraph/python/src/pyngraph/node_factory.cpp (hunk 3 above)
Comment (itikhono) [reply ngraph/python/src/pyngraph/node_factory.cpp:78]: done

File: ngraph/python/src/ngraph/opset8/ops.py (hunk 4 above)
Comment (itikhono) [reply ngraph/python/src/ngraph/opset8/ops.py:112]: I'm not sure it's a good idea, we have 2 separate constructors in ngraph (with/without mask input), so we do not provide any values in mask unless it was provided by user itself. It helps us to avoid unnecessary multiplication.

Comment (jiwaszki) [reply ngraph/python/src/ngraph/opset8/ops.py:112]: Discussed offline

Comment (itikhono) [reply ngraph/python/src/ngraph/opset8/ops.py]: done

**Review by jiwaszki — APPROVED** [review]
LGTM

**Review by akuporos — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]
